src/pages/auth/ForgotPassword.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const ForgotPassword = () => {
+  return (
+    <>
+      <h1>Hello ForgotPassword</h1>
+    </>
+  );
```

Comment:
```suggestion
    <h1>Hello ForgotPassword</h1>
  );
```

Dùng component Typography của material ui https://mui.com/material-ui/react-typography/